### PR TITLE
Prime jedi conversion

### PR DESCRIPTION
**Pull request by bryanfoley**

### CODE REVIEW

File: libSmeagol/nonVolatilePocket.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class NonVolatilePocket(TimerPocket):
     ## Initializes a thread for the registry to handle the automatic saving
     #  @param filename The filename to read/write the preferences (absolute path starts with '/')
     #  @param save_interval The minimal interval between saving of changes, with a default of 5 seconds
-    def __init__(self, filename: str, save_interval: int=5) -> None:
+    def __init__(self, filename: str, save_interval: int = 5) -> None:
```
Comment (robinmdh) [inline libSmeagol/nonVolatilePocket.py:22]: As far as I know this is not valid PEP8 style.
"Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter" from https://www.python.org/dev/peps/pep-0008/#other-recommendations

Comment (bryanfoley) [reply libSmeagol/nonVolatilePocket.py]: Corrected.

**Comment by jackha** [comment]
It doesn't have a release.sh :-(

**Comment by bryanfoley** [comment]
> It doesn't have a release.sh :-(

Yeah, the last piece of the puzzle and I forgot all about it until Jack pointed it out.

**Comment by jackha** [comment]
- Checked package contents
- Checked cloudsmith dev package release
- Checked local build
- Tested on machine

**Pull request merged**